scripts/superdesk/directives/sdFiletypeIcon.js

```diff
@@ -1,7 +1,16 @@
 export default angular.module('superdesk.directives.filetypeIcon', [])
+/**
+ * sdFiletypeIcon adds proper "filetype-icon-*" class based item.
```

Comment:
I wanted to write item type, but there other checks like `item.highlight`, etc. But I'll add "the", I'm not perfect in grammar :).